packages/elements/src/datetime-field/types.ts

```diff
@@ -0,0 +1,14 @@
+export type NavigationKeys = 'ArrowLeft' | 'ArrowRight' | 'ArrowUp' | 'ArrowDown' | 'Escape';
```

Comment:
we should, I will add

Reply:
'Home' and 'End' do not fully work. The issue is when the value does not fit the input, so we might end-up with the situation when selected part is outside scrollable area. And I really do not want to calculate scrollLeft based on CSS font size.

Reply:
Yeah I thought so too. Date time parts are dynamic and it would not benefit much anyway given that maximum number of parts is acceptable. Let's leave it as it is.